epitator/count_annotator_2.py

```diff
@@ -0,0 +1,209 @@
+from epitator.annodoc import AnnoDoc
+from epitator.annospan import AnnoSpan, SpanGroup
+from epitator.annotier import AnnoTier
+from epitator.annotator import Annotator
+from epitator.spacy_annotator import SpacyAnnotator, SentSpan, TokenSpan
+from epitator.spacy_nlp import spacy_nlp
+from epitator.utils import parse_spelled_number, flatten, merge_dicts
+from lazy import lazy
+
+
+class CountSpan(AnnoSpan):
+    def __init__(self, span, metadata):
+        self.start = span.start
+        self.end = span.end
+        self.doc = span.doc
+        self.label = span.text
+        self.metadata = metadata
+
+    def to_dict(self):
+        result = super(CountSpan, self).to_dict()
+        result.update(self.metadata)
+        result['text'] = self.text
+        return result
+
+
+def parse_count_text(count_text, verbose=False):
+    try:
+        count = int(count_text)
+    except ValueError:
+        pass # Try to parse it as a float
+    try:
+        count = float(count_text)
```

Comment:
Not sure we need to handle floating point counts. I can't think of any instances of fractional cases/deaths. Things like "1.5 million" need to be handled by the parsed_spelled_number routine.